tests/serialized_test_data_generation/configs/c96_6ranks_baroclinic.yml

```diff
@@ -0,0 +1,327 @@
+data_table: default
+diag_table: gs://vcm-fv3config/config/diag_table/baroclinic_test/v1.0/diag_table
+field_table: gs://vcm-fv3config/config/field_table/baroclinic_test/v1.0/field_table
+experiment_name: c96_6ranks_baroclinic
+forcing: gs://vcm-fv3config/data/base_forcing/v1.1/
+orographic_forcing: gs://vcm-fv3config/data/orographic_data/v1.0
+# To use internal test cases, point to an empty ICs directory
+initial_conditions: ""
+namelist:
+  amip_interp_nml:
+    data_set: hurrell
+    date_out_of_range: climo
+    do_sst_pert: false
+    interp_oi_sst: true
+    no_anom_sst: true
+    sst_pert: 0.0
+    sst_pert_type: random
+    use_daily: false
+    use_ncep_ice: true
+    use_ncep_sst: true
+  atmos_model_nml:
+    blocksize: -1
+    chksum_debug: false
+    dycore_only: true
+    fdiag: 1.0
+  cires_ugwp_nml:
+    knob_ugwp_azdir:
+    - 2
+    - 4
+    - 4
+    - 4
+    knob_ugwp_doaxyz: 1
+    knob_ugwp_doheat: 1
+    knob_ugwp_dokdis: 0
+    knob_ugwp_effac:
+    - 1
+    - 1
+    - 1
+    - 1
+    knob_ugwp_ndx4lh: 4
+    knob_ugwp_solver: 2
+    knob_ugwp_source:
+    - 1
+    - 1
+    - 1
+    - 0
+    knob_ugwp_stoch:
+    - 0
+    - 0
+    - 0
+    - 0
+    knob_ugwp_version: 0
+    knob_ugwp_wvspec:
+    - 1
+    - 32
+    - 32
+    - 32
+    launch_level: 55
+  cloud_diagnosis_nml:
+    reiflag: 4
+    reimax: 150.0
+    reimin: 10.0
+    rewmax: 10.0
+    rewmin: 5.0
+  coupler_nml:
+    atmos_nthreads: 1
+    calendar: julian
+    current_date:
+    - 2016
+    - 8
+    - 1
+    - 0
+    - 0
+    - 0
+    days: 0
+    dt_atmos: 225
+    dt_ocean: 225
+    hours: 0
+    memuse_verbose: false
+    minutes: 3
+    months: 0
+    ncores_per_node: 32
+    seconds: 45
```

Comment:
How many timesteps do we want these configurations to specify? For the baseline run, these will have to change... but we probably don't want 60 timesteps or whatever here running in jenkins... I wonder how long that takes. Are there any other namelist parameters that would have to change for the performance baseline and comparison runs?